src/assets/icon/PostToCommunity.js

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import Svg, {Path, Rect} from 'react-native-svg';
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good addition of PropTypes to validate the props for the `PostToCommunity` component. This will help catch bugs related to incorrect prop types.
```diff
+ import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->